website/src/_posts/2021-12-2.1-2.3.md

```diff
@@ -29,38 +29,38 @@ Uppy now has a new `@uppy/audio` plugin for recording audio memos, sounds, Jingl
+It used to work by downloading the requested file from a remote provider on the server to then start uploading it to the destination. This did the job, but it’s wasn’t ideal.
 
-We now build streaming support into Companion, which means the download and upload can happen simultaneously. This can result in **2x times faster uploads** because we no longer need to wait for files to finish downloading before starting the upload process. It’s especially useful for large files and removes the need for a lot of storage space on the server, as data will be buffered in-memory.
+We now built streaming support into Companion, which means the download and upload can happen simultaneously. This can result in **two times faster uploads**, because we no longer have to wait for files to finish downloading before starting the upload process. It’s especially useful for large files and removes the need for a lot of storage space on the server, as data will be buffered in memory.
 
 You can enable this by setting `streamingUpload` in the Companion [options](https://uppy.io/docs/companion/#Options).
 
-**NOTE:** This is turned off by default. Do not turn it on yet if you’re using TUS for uploads, due to [this tus-js-client bug](https://github.com/tus/tus-js-client/issues/275).
+**NOTE:** Streaming is turned off by default. Do not turn it on yet if you’re using TUS for uploads, due to [this tus-js-client bug](https://github.com/tus/tus-js-client/issues/275).
 
-Interested in using Companion? You can use Transloadit hosted Companion on every [plan](https://transloadit.com/pricing/).
+Interested in using Companion? You can use Transloadit-hosted Companion on every [Plan](https://transloadit.com/pricing/).
 
-## Status bar error state improvements
+## Status Bar error state improvements
 
-Status bar plugin would get confused about upload errors in Uppy, and we’ve addressed that:
+The Status Bar plugin would get confused about upload errors in Uppy, and we’ve addressed this:
 
-* Error state is shown if only _one_ failed file instead of _all_ failed files. Used to be stuck in the uploading state when a file failed.
-* Added “x of x files uploaded” below “Upload failed” for extra context.
-* Improved the error details button styling in the statusbar and the file info card.
-* Status bar state is set to complete if the user manually removes the failed files.
+* The error state is now shown in case of only _one_ failed file instead of _all_ failed files. The plugin used to get stuck in the uploading state when a single file failed.
+* Added “X of X files uploaded” below “Upload failed” for extra context.
+* Improved the error details button styling in the Status Bar and the file info card.
+* Status Bar state is set to complete if the user manually removes the failed files.
 
 ![status bar improvements screenshot](/images/blog/2.1-2.3/status-bar-improvements.jpg)
 
 ## Internal housekeeping
 
-### Yarn v3 instead of npm
+### Yarn v3 instead of NPM
```

Comment:
I don't think this is correct, npm themselves always use it all lowercase (e.g. https://docs.npmjs.com/about-npm)

Reply:
Ah alright. This was an edit for consistency. The second paragraph of this blog contains NPM written in upper case, so I just followed that example here.